sweepbatcher/sweep_batcher_presigned_test.go

```diff
@@ -59,6 +63,16 @@ func (h *mockPresignedHelper) SetOutpointOnline(op wire.OutPoint, online bool) {
 	h.onlineOutpoints[op] = online
 }
 
+// SetOutpointOnline changes the online status of an outpoint.
+func (h *mockPresignedHelper) SetChangeForPrimaryDeposit(op wire.OutPoint,
```

Comment:
I propose to make this method private so it is clear that it is not a part of public interface but used for testing purposes.

Reply:
fixed